projects/pre-sentence-reports-to-delius/tech-docs/source/async-api-reference.html.md.erb

```diff
@@ -0,0 +1,15 @@
+---
+title: AsyncAPI Reference
+source_url: 'https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/projects/pre-sentence-reports-to-delius/tech-docs/source/async-api-reference.html.md.erb'
+weight: 30
+---
+
+<header id="api" class="anchored-heading govuk-heading-xl" style="padding-top: 30px">
+    <a href="#api" class="anchored-heading__icon" aria-hidden="true" tabindex="-1"></a>
+    Async API Reference
+</header>
+<style>.technical-documentation > h1 { display: none !important }</style>
+
+<h1>ASyncAPI</h1>
```

Comment:
You don't need this bit, it was a hack to get around the tech-docs OpenAPI parser messing with the header.  You should be able to just use Markdown as normal.

```suggestion
# Async API Reference
```